jenkins-scripts/lib/windows_library.bat

```diff
@@ -268,8 +268,20 @@ if NOT %VCPKG_HEAD% == %VCPKG_TAG% (
 goto :EOF
 
 :: ##################################
-:install_vcpkg_package
-:: arg1: package to install
+:remove_vcpkg_installation
+:: remove the installed directory to simulate all packages removal
+:: vcpkg cli does not support the operation
+set LIB_DIR="%~dp0"
+call %LIB_DIR%\windows_env_vars.bat || goto :error
+if [%VCPKG_INSTALLED_FILES_DIR%]==[] (
+  echo VCPKG_INSTALLED_FILES_DIR seems empty, refuse to delete
+  goto :error

```

Comment:
Are there situations where we'd expect this directory to be empty, such as with a freshly provisioned host, and if so is the `goto :error` recoverable or would this halt the build?